src/olympia/promoted/management/commands/promote_by_firefox_themes.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from olympia import amo
+from olympia.addons.models import Addon
+from olympia.constants.promoted import LINE
```

Comment:
We are planning to deprecate the classes after we finish multiple promoted groups. Please use the chocies object instead. `PROMOTED_GROUPS_BY_ID` keeps the access centralized so it will be easier to clean up.